README.md

```diff
@@ -12,9 +12,34 @@ This module requires the following modules/libraries:
 ## Installation
 
   1. Install as usual, see [this](https://drupal.org/documentation/install/modules-themes/modules-7) for further information.
-  1. Ensure that your xslt used by gsearch will index the ancestor_ms field.  If you are using discoverygarden's [basic-solr-config](https://github.com/discoverygarden/basic-solr-config?source=c), this [gist](https://gist.github.com/daniel-dgi/6001819) will point you in the right direction.
-  1. Set up the 'Islandora Collection Search' block. It can replace the standard 'Islandora simple search' block.
-  
+  2. Ensure that your xslt used by gsearch will index the `ancestors_ms` field.  If you are using discoverygarden's [basic-solr-config](https://github.com/discoverygarden/basic-solr-config?source=c), get your System Administrator to make the following changes in your `foxmlToSolr.xslt`:
+      1. Find the section that looks like [this](https://github.com/discoverygarden/basic-solr-config/blob/modular/foxmlToSolr.xslt#L27-L30), and make it look like this:<br>
+      `xmlns:dgi-e="xalan://ca.discoverygarden.gsearch_extensions"`<br>
+      `xmlns:sparql="http://www.w3.org/2001/sw/DataAccess/rf1/result"`<br>
+      `xmlns:xalan="http://xml.apache.org/xalan">`<br>
+      2. Find the section that looks like [this](https://github.com/discoverygarden/basic-solr-config/blob/modular/foxmlToSolr.xslt#L114-L120), and uncomment the code. Leave the comments as comments.
+      3. Find the section that looks like [this](https://github.com/discoverygarden/basic-solr-config/blob/modular/foxmlToSolr.xslt#L277-L287), and uncomment it.
+      4. After making these changes, fedora will need to be restarted, and re-indexed.
+  3. Set up the 'Islandora Collection Search' block. It can replace the standard 'Islandora simple search' block.
+
+## Configuration
+
+### Configuring the Module
+
+The configuration page for the module can be found at `yoursite.com/admin/islandora/tools/collection_search`.
+The options in the 'Collections' section of that page provide ways to configure the UI, such as having a drop-down menu of certain collections that can be searched from anywhere on your site where the 'Collection Search' block is enabled.
+You shouldn't need to change the 'Ancestor field' or 'GSearch Config'.
```

Comment:
End users will almost certainly need to set GSearch Config.  It contains creds that should not be default on production.